libs/init/migrate/api/src/mill/init/migrate/ModuleConfig.scala

```diff
@@ -0,0 +1,108 @@
+package mill.init.migrate
```

Comment:
How does this file related to the code we already have in libs/init/buildgen/src/mill/main/buildgen/ir.scala?